public/js/overrides.js

```diff
@@ -47,6 +47,17 @@ Garp.mailtoOrUrlPlugin = {
 };
 
 /**
+ * Override email VType. It otherwise doesn't support .amsterdam domainnames
+ *
+ * @TODO: decide if we need to implement the real list
+ * @see: http://data.iana.org/TLD/tlds-alpha-by-domain.txt
```

Comment:
Waarom halen we die ",19" niet gewoon helemaal weg?  
Dan zijn we nog robuuster toch? Dan staan we alles toe dat groter of gelijk aan 2 karakters is. Lijkt mij prima en dan hoeven we er nooit meer naar te kijken 🙂